### DISCUSSION TITLE
New to Chia...started script...12 hours later computer crashed... found debug file...errors everywhere...Please help!

### DISCUSSION THREAD

**Discussion by hereis2u** [Chia-Network/chia-blockchain#1180] [Q&A]
Hello
I have no programing or IT experience.  Chia fascinated me so I bought a gaming PC just to try out Chia!  I was 4 days post surgery and have never built a PC so I paid a local man to put together.  Finally  got the machine started and rand the necessary scripts.  Things were humming along well but after 12 hours, still no plots on my hard drives.  My computer has 2 SSD drives and 5 HDDs (three 14TBs and two 12TBs).  Currently have 32GB of RAM with an AMD RYZEN processor.  
here is how I set up my drives: 

![image](https://user-images.githubusercontent.com/80245561/110290862-7dfe8e80-7fa8-11eb-9c67-2652a01afd69.png)

Here were some snapshots before all my command windows closed. :-(

![IMG_2952](https://user-images.githubusercontent.com/80245561/110290570-26f8b980-7fa8-11eb-8689-f46b7c876cdf.jpg)

Please zoom in on the last 3 lines of the screenshot above.  It says something about only writing xxxx of yyyy and error and will try in 5 minutes.  

after the crash, I started snooping around and found a debug file and it had errors all over it.  

here is the debug file: 

[debug1.txt](https://github.com/Chia-Network/chia-blockchain/files/6099712/debug1.txt)

1.  What am I missing? 
2. Do I need to stop the process, turn off my computer, delete the temp files in my SSD drives and start all over again? 

Any help would be appreciated

thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
